entity-management-mongo/src/main/java/eu/europeana/entitymanagement/mongo/repository/EntityRecordRepository.java

```diff
@@ -186,6 +186,13 @@ public List<EntityRecord> findWithFilters(int start, int count, Filter[] filters
         .toList();
   }
 
+  public List<EntityRecord> findAll(int start, int count) {
```

Comment:
what is still method needed for?

Reply:
it is used in the reindexSolrFromMongo() test to reindex solr from mongo (although the test is not executed automatically)